jumpstart-week2/README.md

```diff
@@ -0,0 +1,222 @@
+}
+```
+
+#### Selectors
+
+CSS provides about a dozen ways of defining the elements you want a rule to apply to. We'll cover 3 of the most basic. You can generally get by with these for most use cases, but if a situation requires more complex selection logic you should be aware that other methods exist and you can google them.
+
+The first type of selector is an element tag. You simply write the name of an HTML element and all elements of that type will be selected.
+
+The next two types of selectors are looking for HTML elements with particular attribute values. Every HTML element can optionally define an attribute called 'id' and attribute called 'class'. We can then select an element with a particular id value by preceding the id value with a `#` and we can select an element with a particular class value by preceding the value with a `.`.
+
+The only difference between the two is that an element's id value should be unique, no other element on the page should have the same id value. Many elements can have the same class value however. Generally, you should favor using classes rather than ids because it makes your code more reusable.
+
+For the rest of the CSS selectors, see [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors).
+
+
+#### Properties
+
+> Google is your best friend
+
+CSS properties allow you to do just about anything to HTML elements. The three big concepts you'll be doing with CSS in order of complexity are adding style/color to text or elements, changing the size of elements, and defining the layout of your elements.
+
+##
+
+Styling element's color or text is often a trial of finding the right attribute for your use case. The fastest way to get better at CSS is by trying to write HTML and CSS webpages and using google to find the right techniques for your situation.
+
+To make things even more confusing, for each property there can be multiple ways of expressing the same value. The `color` property lets you specify an element's color, and it takes a color value. A color value can be just the name of the color ('red', 'blue', 'gray'), an RGB value ('rgb(255, 0, 255)', 'rgb(0, 0, 255)', 'rgb(125,125,125)'), a Hex color ('#FF0000', '#0000FF', '666666'), or just about any other way you can [define a color](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value).
+
+##
+
+To tackle sizing and positioning of elements, you first have to understand the "box model".
+
+Every HTML element has several boxes around it whose dimensions can be controlled through CSS. Every element has all of these boxes around it, though often these boxes can coincide.
+
+The innermost box is the "content box". This is the space that all the children of that element occupy. 
+
+The next box is the "border box". This includes all the content of the content box PLUS the border of the element.
+
+Then is the "padding box". This includes the border box PLUS some padding (empty space) that the element should be surrounded by.
+
```

Comment:
Throw in an image like ![box model](https://www.kasandbox.org/programming-images/misc/boxmodel.png)